src/mmrelay/plugins/weather_plugin.py

```
@@ -903,56 +914,66 @@ async def handle_room_message(
+                units = WEATHER_UNITS_METRIC
+            marine = await asyncio.to_thread(
+                self.generate_marine_forecast,
+                coords[0],
+                coords[1],
+                parsed_command,
+                units,
+            )
+            if marine:
+                full_forecast = f"{terrestrial} | {marine}"
+            else:
+                full_forecast = terrestrial
```

Ruff (0.15.10)

[warning] Use ternary operator `full_forecast = f"{terrestrial} | {marine}" if marine else terrestrial` instead of `if`-`else`-block

Replace `if`-`else`-block with `full_forecast = f"{terrestrial} | {marine}" if marine else terrestrial`

(SIM108)